src/utils/DDOHelpers.ts

```diff
@@ -9,6 +9,11 @@ import {
 import { AssetPrice } from '../models'
 import { BigNumber } from './BigNumber'
 
+// DDO Services including a sales process
+const SALES_SERVICES = ['nft-sales', 'access', 'compute', 'nft-sales']
```

Comment:
nft-sales is repeated twice. Is that correct?

Reply:
you are right, fixing it